plugins/modules/k8s_cluster_info.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020, Abhijeet Kasurde
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+module: k8s_cluster_info
+
+version_added: "0.11.1"
+
+short_description: Describe Kubernetes (K8s) API available and their respective versions
+
+author:
+    - Abhijeet Kasurde (@Akasurde)
+
+description:
+  - Use the OpenShift Python client to perform read operations on K8s objects.
+  - Authenticate using either a config file, certificates, password or token.
+  - Supports check mode.
+
+extends_documentation_fragment:
+  - community.kubernetes.k8s_auth_options
+
+requirements:
+  - "python >= 2.7"
+  - "openshift >= 0.6"
+  - "PyYAML >= 3.11"
+'''
+
+EXAMPLES = r'''
+- name: Get an existing Service object
+  community.kubernetes.k8s_cluster_info:
+  register: api_status
+'''
+
+RETURN = r'''
+apis:
+  description:
+  - The API(s) that exists in dictionary
+  returned: success
+  type: dict
+  contains:
+    api_version:
```

Comment:
I would nix `group_version` and make `api_version` `DynamicClient.group_version`. It's a bit confusing, but to users `apiVersion` means `{group}/{api_version}`, but internally most clients deal with them separately.